code/game/objects/items/storage/boxes.dm

```diff
@@ -561,6 +561,28 @@
 	for(var/i in 1 to 5)
 		new /obj/item/explosive/mine/pmc(src)
 
+/obj/item/storage/box/explosive_mines/satchel_charges
+	name = "\improper M17 satchel charge box (x5)"
+	desc = "A secure box holding five M17 satchel charges."
```

Comment:
![Screenshot 2024-08-12 211532](https://github.com/user-attachments/assets/cf39977c-fea2-4d27-97cd-cb4a896a1fc3)
either bump up the max_storage_space var or decrease amount of charges